waiter/src/waiter/process_request.clj

```diff
@@ -242,22 +242,29 @@
        :fold-chunked-response? true
        :fold-chunked-response-buffer-size output-buffer-size
        :follow-redirects? false
-       :idle-timeout idle-timeout})))
+       :idle-timeout idle-timeout
+       :cookies cookies})))
 
 (defn make-request
   "Makes an asynchronous http request to the instance endpoint and returns a channel."
-  [http-client make-basic-auth-fn instance {:keys [body request-method] :as request}
+  [http-client make-basic-auth-fn instance {:keys [body cookies request-method] :as request}
    {:keys [initial-socket-timeout-ms output-buffer-size]} passthrough-headers end-route app-password metric-group]
   (let [instance-endpoint (scheduler/end-point-url instance end-route)
         service-id (scheduler/instance->service-id instance)
         ; Removing expect may be dangerous http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html, but makes requests 3x faster =}
         ; Also remove hop-by-hop headers https://www.w3.org/Protocols/rfc2616/rfc2616-sec13.html#sec13.5.1
-        headers (-> (dissoc passthrough-headers "expect" "authorization")
+        headers (-> (dissoc passthrough-headers "expect" "authorization" "cookie")
```

Comment:
Nitpick: I know the ordering was already wrong, can we order the keys `"authorization" "cookie" "expect"` now?